cmd/orgIdColumnPopulator/main.go

```diff
@@ -0,0 +1,197 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	_ "github.com/lib/pq"
+
+	"github.com/RedHatInsights/tenant-utils/pkg/orgIdColumnPopulator"
+
+	"github.com/RedHatInsights/tenant-utils/pkg/tenantid"
+	clowder "github.com/redhatinsights/app-common-go/pkg/api/v1"
+
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	cmd := NewRootCommand()
+	if err := cmd.Execute(); err != nil {
+		os.Exit(1)
+	}
+}
+
+func NewRootCommand() *cobra.Command {
+
+	var dbConnMetadata dbConnectionMetadata
+	var dbTable string
+	var dbAccountColumn string
+	var dbOrgIdColumn string
+	var batchSize int
+	var clowderConfig bool
+	var prometheusPushGatewayAddr string
+	var translatorServiceAddr string
+	var translatorServiceTimeout int
+	var dbOperationTimeout int
+
+	var rootCmd = &cobra.Command{
+		Use:   "orgID-column-populator",
+		Short: "Run the org_id column population util",
+		Run: func(cmd *cobra.Command, args []string) {
+
+			var err error
+
+			if clowderConfig == true {
+				dbConnMetadata, err = readDatabaseConfigFromClowder()
+				if err != nil {
+					log.Fatal(err)
+				}
+			}
+
+			connStr := buildDBConnectionString(dbConnMetadata)
+
+			db, err := initializeDBConnection(connStr)
+
+			accountNumberTranslator := buildAccountNumberTranslator(translatorServiceAddr, translatorServiceTimeout)
+
+			metricsRecorder := buildMetricsRecorder(prometheusPushGatewayAddr)
+
+			err = orgIdColumnPopulator.MapAccountToOrgId(context.TODO(), db, dbTable, dbAccountColumn, dbOrgIdColumn, dbOperationTimeout, batchSize, accountNumberTranslator, metricsRecorder)
+			if err != nil {
+				log.Fatal(err)
+			}
+		},
+	}
+
+	rootCmd.Flags().StringVarP(&dbConnMetadata.host, "db-host", "l", "localhost", "Database hostname")
+	rootCmd.Flags().IntVarP(&dbConnMetadata.port, "db-port", "p", 5432, "Database port")
+	rootCmd.Flags().StringVarP(&dbConnMetadata.user, "db-user", "u", "insights", "Database user")
+	rootCmd.Flags().StringVarP(&dbConnMetadata.password, "db-password", "w", "1234", "Database password")
+	rootCmd.Flags().StringVarP(&dbConnMetadata.name, "db-name", "n", "cloud-connector", "Database name")
+	rootCmd.Flags().StringVarP(&dbConnMetadata.sslMode, "db-ssl-mode", "s", "disable", "Database ssl mode")
```

Comment:
I'd rather default to `verify-full` and do the extra work if I need to disable ssl